pykusto/expressions.py

```diff
@@ -96,6 +96,13 @@ def has(self, exp: StringType) -> 'BooleanExpression':
         # The pattern for the search expression must be a constant string.
         return BooleanExpression(KQL(f'{self.kql} has {to_kql(exp)}'))
 
+    def has_cs(self, exp: StringType) -> 'BooleanExpression':
+        """
+        https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/datatypes-string-operators
+        """
+        # The pattern for the search expression must be a constant string.
+        return BooleanExpression(KQL(f'{self.kql} has_cs {to_kql(exp)}'))
```

Comment:
We should use `as_subexpression` and `_subexpr_to_kql`, to prevent operation precedence issues

Reply:
_subexpr_to_kql is used in two functions (any_of, all_of), and as_subexpression in none, we can change it, but I think it's a different PR

Reply:
I don't understand you answer. See what I meant in #109